frontend/test/unit/specs/utils/attribution-html.spec.ts

```diff
@@ -62,6 +62,17 @@ describe("getAttribution", () => {
     expect(attrText).toContain(expectation)
   })
 
+  it("escapes embedded HTML", () => {
+    const mediaItemWithHtml = {
+      ...mediaItem,
+      originalTitle: '<script>console.log("HELLO");</script>',
+    }
+    const attrText = getAttribution(mediaItemWithHtml, i18n)
+    const expectation =
+      "&lt;script&gt;console.log(&quot;HELLO&quot;);&lt;/script&gt;"
+    expect(attrText).toContain(expectation)
```

Comment:
Maybe also this, for good measure?

```suggestion
    expect(attrText).toContain(expectation)
    expect(attrText).not.toContain(mediaItemWithHtml.originalTitle)
```